src/AzureStack/InfrastructureInsightsAdmin/InfrastructureInsights.Admin/Microsoft.AzureStack.Management.InfrastructureInsights.Admin.csproj

```diff
@@ -1,18 +1,27 @@
-﻿
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
     <!-- Please do not move/edit code below this line -->
     <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
     <!-- Please do not move/edit code below this line -->
 
-  <PropertyGroup>
-    <PackageId>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</PackageId>
-    <Description>Provides developers with libraries for the health admin under AzureStack to manage health alerts.</Description>
-    <VersionPrefix>0.1.0-preview</VersionPrefix>
-    <AssemblyName>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</AssemblyName>
-    <PackageTags>Microsoft Azure Stack;InfrastructureInsights;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
-    <PackageReleaseNotes>This is a preview release of the AzureStack InfrastructureInsights Admin SDK. Included with this release is support for gathering information about the health of your AzureStack infrastructure.</PackageReleaseNotes>
-  </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-  </PropertyGroup>
+    <PropertyGroup>
+        <PackageId>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</PackageId>
```

Comment:
Looks like this package is already published and there are some additive changes being introduced. Please update the package version number